### ISSUE TITLE
Error using --list without metric argument

### ISSUE THREAD

**Issue by ravenich** [NagiosEnterprises/ncpa#113]
Trouble with two slashes in connecting string after api word
./check_ncpa.py -H 192.168.0.233 -t mytoken --list -v
Connecting to: https://1.1.1.1:5693/api//?token=mytoken
File returned contained:
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.</p>

An error occurred:No JSON object could be decoded

**Comment by varnar** [comment]
Confirmed - two slashes return me also same result - 404
executing same with one slash after "api" word works as expected.

I make changes in function ```def get_host_part_from_options(options):```
Instead of just ``` 'https://%s:%d/api/%s/%s' % (hostname, port, metric, arguments)```
I put following:
```
    if metric is '' and arguments is '':
        api_address = 'https://%s:%d/api' % (hostname, port)
    else:
        api_address = 'https://%s:%d/api/%s/%s' % (hostname, port, metric, arguments)
```

**Comment by varnar** [comment]
Please re-open it, so developers can make them as BUG and fix the script.

Thanks

**Comment by scot0357** [comment]
Good catch @varnar. I made slight mods but otherwise your fix was exactly what was needed. I changed the version on the plugin and it will soon be merged into master.
